tiledb/libtiledb.pyx

```diff
@@ -1330,22 +1330,9 @@ cdef class Array(object):
                     timestamp_end
             )
         else:
-            array_instance = self_or_uri
-            warnings.warn(
-                "The `tiledb.Array.delete_fragments` instance method is deprecated. Use the static method with the same name instead.",
-                DeprecationWarning,
-            )
-            ctx_ptr = safe_ctx_ptr(array_instance.ctx)
-            array_ptr = <tiledb_array_t*>array_instance.ptr
-            buri = array_instance.uri.encode('UTF-8')
-
-            rc = tiledb_array_delete_fragments(
-                    ctx_ptr,
-                    array_ptr,
-                    buri,
-                    timestamp_start,
-                    timestamp_end
-            )
+            # TODO: Make this method static and entirely remove the conditional.
```

Comment:
How about we do this now?

Reply:
Do you mean to reimplement `tiledb.delete_fragments()`?
It was there before https://github.com/TileDB-Inc/TileDB-Py/pull/1966

Reply:
Yes, we could essentially revert #1966. What do you think?

Reply:
But `tiledb.delete_fragments` was supposed to be deprecated and the users were forced to use `Array.delete_fragments`. cc. @nguyenv